packages/main/src/TableCellBase.ts

```diff
@@ -31,6 +32,14 @@ abstract class TableCellBase extends UI5Element {
 	@property({ type: Boolean })
 	_popin = false;
 
+	/**
+	 * Determines the horizontal alignment of table cells.
+	 * @default "Start"
+	 * @public
+	 */
+	@property()
+    horizontalAlign?: `${TableCellHorizontalAlign}`;
```

Comment:
I would expect
`horizontalAlign: `${TableCellHorizontalAlign}` = "Start";`

Reply:
Well I used hAlign but got a comment about the name. https://github.com/SAP/ui5-webcomponents/pull/9639#discussion_r1738100712

Technically  the default value should be "Normal" now because that's the default value if justify-content is not set. But it behaves like "Start". It is this way now because I've changed the code so that the CSS variables are only created if horizontalAlign is used on the headerCell. Take a look at this comments https://github.com/SAP/ui5-webcomponents/pull/9639#discussion_r1740179899

Reply:
the consistent name horizontalAlign instead of hAlign is OK.

but the default value is still technically `undefined` isn't it?

Reply:
Yes you're right. Since `horizontalAlign` is undefined, `justify-content` will get the default value `normal`.

Reply:
As discussed I've changed it. Since other components use `@default undefined`, I've changed my code accordingly.